student.rb

```diff
@@ -1,19 +1,14 @@
-require './person'
+require_relative './person'
 
 class Student < Person
-  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission: true)
-    @age = age
-    @name = name
+  attr_accessor :classroom
+
+  def initialize(classroom, name, age, parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly create a method that adds a single student to the classroom's students array. This will serve as the `belong_to` side of the relationship. 👍 You can read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for a better understanding. 👍